### ISSUE TITLE
Sometime, we can get stuck in the New Offer page

### ISSUE THREAD

**Issue by Ganbin** [xch-dev/sage#125]
When creating offer, it happen that we can get stuck in the New Offer page.
It is then impossible to go back to the main Offers Page.

To reproduce:
1. Create a new Offer, with any assets.
2. Click Create Offer Button.
3. Instead of Saving the offer, click outside or click the `x` button to close the modal dialog.

The issue with that is that we will lost the offer in the wallet because it is not saved. But we could have shared it externally (on Dexie for example 😄). We then need to import it again if we want to see it on the wallet.

Really looking forward for this amazing wallet that really is blazing fast.

Version `0.8.1` - Mac OS Sonoma

**Comment by Rigidity** [comment]
Thanks, I've definitely noticed things getting stuck on the offer page. I'll look into it

**Comment by Rigidity** [comment]
Is this fixed @Ganbin?

**Comment by Ganbin** [comment]
Yep, I confirm it is now working. Clicking the `x` button or outside of the modal doesn't make us stack in the New Offer page. We can correctly go back and the offer is correctly saved.

Thanks for the fix.
